gc/base/MemoryPoolAddressOrderedList.cpp

```diff
@@ -698,6 +739,37 @@ MM_MemoryPoolAddressOrderedList::internalAllocateTLH(MM_EnvironmentBase *env, ui
 	return false;
 }
 
+uintptr_t
+MM_MemoryPoolAddressOrderedList::getConsumedSizeForTLH(MM_EnvironmentBase *env, MM_HeapLinkedFreeHeader* freeEntry, uintptr_t maximumSizeInBytesRequired) {
```

Comment:
move star
MM_HeapLinkedFreeHeader *freeEntry

Reply:
opps missed this one